YARN_MIGRATION.md

```diff
@@ -0,0 +1,23 @@
+# YARN 2 MIGRATION DOC
+
+## Remaining todos:
+
+### fix all missing dependencies
+
+to see problems, run `yarn dlx @yarnpkg/doctor`. note: `yarn global` no longer exists. instead, yarn has the equivalent of `npx` (roughly) via `yarn dlx`.
```

Comment:
just tried this out - this feature is pretty cool! We have some errors ;-;-;-;

Reply:
haha yea :(